### PR TITLE
Marni wip 219 forget confirmation

### PR DESCRIPTION
**Pull request by marnibrewster**
First attempt at creating a confirmation modal to appear when a user clicks "Forget" to remove an account.

### CODE REVIEW

File: packages/app-accounts/src/Editor.tsx [hunk 1]
```diff
@@ -38,8 +40,16 @@ class Editor extends React.PureComponent<Props, State> {
...
   render () {
+    const { t } = this.props;
+    const { isForgetOpen } = this.state;
     return (
       <div className='accounts--Editor'>
+        <Forgetting
+          isOpen={isForgetOpen}
+          onClose={this.toggleForget}
+          doForget={this.onForget}
+          t={t}
```
Comment (jacogr) [inline packages/app-accounts/src/Editor.tsx]: We don't need to pass t through, just wrap the `Forgetting` component export in a `translate`

File: packages/app-accounts/src/Forgetting.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  renderButtons () {
+    const { onClose, doForget } = this.props;
+
+    return (
+      <Modal.Actions>
+        <Button.Group>
+          <Button
+            isNegative
+            onClick={onClose}
+            text='Cancel'
```
Comment (jacogr) [inline packages/app-accounts/src/Forgetting.tsx]: Just warp the text in a `t`, i.e. `t('forget.close', { defaultValue: 'Cancel' })` (As per other comments, import `translate` and then wrap the export)

File: packages/app-accounts/src/Forgetting.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+    return [
+      <Modal.Header key='header'>
+        Are you sure you want to delete this account?
+      </Modal.Header>
+    ];
+  }
+}
+
+export default Forgetting;
```
Comment (jacogr) [inline packages/app-accounts/src/Forgetting.tsx]: Just warp with a `translate` to allow the HOC to inject the `t` as a prop. (Don't need to pass it through from the parent)

```
import translate from './translate';
...
export default translate(Forgetting);
```

File: packages/app-accounts/src/Forgetting.tsx [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+        </Button.Group>
+      </Modal.Actions>
+    );
+  }
+
+  renderContent () {
+
+    return [
+      <Modal.Header key='header'>
+        Are you sure you want to delete this account?
```
Comment (jacogr) [inline packages/app-accounts/src/Forgetting.tsx]: Translatable string.

File: packages/app-accounts/src/Forgetting.tsx [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+            text='Delete'
+          />
+        </Button.Group>
+      </Modal.Actions>
+    );
+  }
+
+  renderContent () {
+
+    return [
```
Comment (jacogr) [inline packages/app-accounts/src/Forgetting.tsx]: Remove the blank line before. (nitpick)

File: packages/app-accounts/src/Forgetting.tsx [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+type Props = I18nProps & {
+  isOpen: boolean,
+  onClose: () => void,
+  doForget: () => void
+};
+
+class Forgetting extends React.PureComponent<Props> {
+
+  constructor (props: Props) {
```
Comment (jacogr) [inline packages/app-accounts/src/Forgetting.tsx]: Extra blank line before.

**Comment by marnibrewster** [comment]
Thanks for the clear explanation of the mistakes, this is super helpful for me as I learn!

File: packages/app-accounts/src/Forgetting.tsx [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+            onClick={onClose}
+            text={t('forget.close', {
+              defaultValue: 'Cancel'
+            })}
+          />
+          <Button.Or />
+          <Button
+            isPrimary
+            onClick={doForget}
+            text='Delete'
```
Comment (jacogr) [inline packages/app-accounts/src/Forgetting.tsx]: Small oversight :) Translatable.

Comment (marnibrewster) [reply packages/app-accounts/src/Forgetting.tsx]: Workflow question: should I make these changes in this pull request, even though it's already been merged / closed?

Comment (jacogr) [reply packages/app-accounts/src/Forgetting.tsx]: I should probably not have merged with that outstanding - however I marked "all-ok" at night, then tested "all-ok" the next morning ... Not the end of the world, that is a small fix, but it slipped through :)

**Pull request merged**